manifests/monitoring/sensu.pp

```diff
@@ -0,0 +1,18 @@
+# See README.md for details
+class consul_replicate::monitoring::sensu {
+
+  package { 'sensu':
+    ensure => installed
+  }
+
+  file { 'Ensure check-procs.rb is present':
+    ensure => file,
+    path   => '/etc/sensu/plugins/processes/check-procs.rb'
+  }
```

Comment:
Same here.  If we try to create the below file and the sensu conf dir doesn't yet exist, the run will fail.